service/lib/agama/storage/manager.rb

```diff
@@ -107,7 +107,7 @@ def on_probe(&block)
 
       # Probes storage devices and performs an initial proposal
       def probe
-        start_progress(4)
+        start_progress_with_size(4)
```

Comment:
I know in these case the steps are known in advance too, but this progress is only used for the "details". Anyway, I would use the `with_descriptions` constructor. Maybe they would be needed if we change the UI. Anyway, probably you want to change the overall approach of the progress API. So we could keep it as it is for now.

Reply:
Initially, I used the `with_description` constructor, but as it was not really needed, I decided to revert that change.